llvm-examples/dense_div_test.cc

```diff
@@ -0,0 +1,44 @@
+#include <iostream>
+#include "../include/taco.h"
+#include "../include/taco/llvm.h"
+
+using namespace taco;
+
+int main() {
+  // Create formats
+  Format list({Dense});
+
+  // Create tensors
+  Tensor<int32_t> A({3}, list);
+  Tensor<int32_t> B({3}, list);
+  Tensor<int32_t> C({3}, list);
+
+  // Insert data into B
+  B.insert({0}, (int32_t) 4);
+  B.insert({1}, (int32_t) 5);
+  B.insert({2}, (int32_t) 6);
+
+  C.insert({0}, (int32_t) 1);
+  C.insert({1}, (int32_t) 2);
+  C.insert({2}, (int32_t) 3);
```

Comment:
Let's keep the division in the integer domain
```suggestion
  C.insert({0}, (int32_t) 2);
  C.insert({1}, (int32_t) 3);
  C.insert({2}, (int32_t) 4);
```

Reply:
ok!